app/src/main/java/com/project/ianime/screens/manageanime/ManageAnimeFragment.kt

```diff
@@ -186,7 +186,7 @@ abstract class ManageAnimeFragment : BaseFragment() {
         _binding = null
     }
 
-    companion object{
+    companion object {
```

Comment:
using jpg and png right now, can support bmp, but bmp file could be large (file size limit is 1mb for now)